contributing_to_docs/templates/overview_topic_template.adoc

```diff
@@ -1,8 +1,31 @@
 = Overview
 {product-author}
 {product-version}
-:data-uri:
-:icons:
+:icons: font
 :experimental:
 
---Remove this and add a brief introduction here--
+--Start with a clear introduction to the topic and what information customers can expect to find in the topic--
```

Comment:
Is "topic" the right word here? Isn't this an Overview topic that describes what information is covered within the overall category/directory/whatever we're calling the groupings on the left nav?